src/schema/posts.ts

```diff
@@ -2342,6 +2342,19 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         throw new ForbiddenError('Access denied!');
       }
 
+      if (status === SourcePostModerationStatus.Rejected) {
+        if (!rejectionReason) {
+          throw new ValidationError('Rejection reason is required');
+        }
+
+        if (
+          rejectionReason?.toLowerCase()?.includes('other') &&
```

Comment:
should it not be `=== 'other'?

Reply:
Yeah, we can use `===` too. Did not remember if it was `other` or `others` from the FE, and just to ensure it with this one. Although I've checked it now and it is just `other`.